donate/payments/tasks.py

```diff
@@ -26,6 +27,95 @@
 stripe.api_key = settings.STRIPE_API_KEY
 
 
+def mofo_donation_receipt_datetime(ts):
+    # convert unix timestamp to e.g. "Thursday, Feb 11, 2021 at 4:20pm (GMT-08:00)"
+    ds = datetime.utcfromtimestamp(float(ts))
+    return ds - timedelta(hours=8)
+
+
+def mofo_donation_receipt_time_string(ds):
+    """Return the date and time formatted as requested by MoFo"""
+    return ds.strftime("%Y-%m-%d %H:%M")
+
+
+def mofo_donation_receipt_day_of_month(ds):
+    """Return the day of the month"""
+    return ds.strftime("%d")
+
+
+def mofo_donation_receipt_number_format(amount):
+    return f"{float(amount):.2f}"
+
+
+DONATION_RECEIPT_FIELDS = [
+    "created",
+    "currency",
+    "donation_amount",
+    "email",
+    "first_name",
+    "last_name",
+    "recurring",
+    "transaction_id",
+    "card_type",
+    "last_4",
+    "locale",
+    "service",
+    "project",
+]
+# map of incoming field names -> email field names
+DONATION_RECEIPT_FIELDS_MAP = {
+    "card_type": "cc_type",
+    "last_4": "cc_last_4_digits",
+    "locale": "donation_locale",
+    "service": "payment_source",
+}
+
+LANGUAGE_IDS = {
```

Comment:
Can you please add some docs around what this is, how it's derived, where can we go if we need to support additional locales, etc.?

Reply:
Hey @benhohner  I tried to add some comments to walk through what the acoustic functions are doing. 

In terms of the Locale question however, would Pmac be a good point of contact?

Reply:
Yep, if you could ping pmac, he might have some insight that you could add to the documentation

Reply:
We could add something like:
> Language ID's are mapped to Basket IDs. Instead of asking Basket for the IDs, because these are likely to never change, we've opted to hard code these and that allows us to avoid another API request to Basket.

Reply:
Sounds good all! Reaching out to pmac now to see if he has any input. Other than that, I removed the print statements, and implemented all other feedback. So I am re requesting review for our meeting !